### ISSUE TITLE
NVDA is not announcing the text present in the "The textarea readonly attribute" multi line edit section in one go.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#16111]
### Steps to reproduce:
1. Hit on this URL: [W3Schools Tryit Editor](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_textarea_readonly).
2. Turn on Screen Reader NVDA.
3. Tab to "The textarea readonly attribute" field.
4. Now check that NVDA is announcing the complete information present in "The textarea readonly attribute" field or not in one go.
### Actual behavior:

NVDA is not announcing the text present in the "The textarea readonly attribute" multi line edit section in one go. NVDA is announcing some information until web it is announcing after that NVDA is not announcing any information present in "The textarea readonly attribute" field.

### Expected behavior:

When NVDA focus lands on "The textarea readonly attribute" field, NVDA should announce the complete information present in the "The textarea readonly attribute" field.
### NVDA logs, crash dumps and other attachments:

https://github.com/nvaccess/nvda/assets/93735775/69702237-1902-4762-9ad5-14be3ccd7f76

#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2023.3.3 (2023.3.30854)
#### Windows version:
Version: 10.0.23620 Build 23620
#### Other information about your system:
OS Name: Microsoft Windows 11 Enterprise Insider Preview

#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes.
#### If NVDA add-ons are disabled, is your problem still occurring?
NVDA (2023.1)

**Comment by lukaszgo1** [comment]
When reporting multi line edit fields, NVDA always reports only the line where the caret is currently situated. Imagine a multi line edit field containing an entire book for example, you certainly would not want to hear the entire content when tabbing to it. This should be closed as invalid.

**Comment by Adriani90** [comment]
Thanks @lukaszgo1 for the good catch, closing as invalid.
